build/mariadb/files/mariadb-template.xml

```diff
@@ -47,16 +47,15 @@ Copyright 2021 OmniOS Community Edition (OmniOSce) Association.
                    restart_on="none">
             <service_fmri value="svc:/milestone/multi-user" />
         </dependent>
-
         <method_context security_flags="aslr">
-            <method_credential user="$(USER)" group="$(GROUP)" />
+            <method_credential user="$(USER)"
+                               group="$(GROUP)" />
         </method_context>
 
         <exec_method type="method"
                      name="start"
                      exec="%{application/method} %m"
-                     timeout_seconds="60">
-        </exec_method>
+                     timeout_seconds="60"></exec_method>
```

Comment:
I guess this can be further cleaned up to:
```
timeout_seconds="60" />
```

Reply:
This is just the result of running the `tools/tidy-manifests` script which we use to keep manifest format consistent across all the packages. Particularly useful since we all have differing personal preferences for indentation etc..